intake_esm/core.py

```diff
@@ -680,6 +678,10 @@ def to_collection(
             pr         (member_id, time, lat, lon) float32 dask.array<chunksize=(1, 600, 160, 320), meta=np.ndarray>
         """
 
+        # Set the separator to a / for datatree
+        self.sep = '/'
```

Comment:
we should make this change temporary and looking into resetting this to its original value before returning... 

```python
self.sep, old_sep = '/', self.sep
...
...
...


self.sep = old_sep
return ....
````

Reply:
Yeah - that might be a better approach. Currently within datatree, I could not find a clean way of extracting the path to the datasets ("A/B/C/.."), and calling catalog.keys() was the only way to grab these strings nicely, and pass that into datatree to subset one of the datasets. For example:

```python
tree = collection.to_datatree()
ds = tree[collection.keys()[0]].ds
```